python/rapidsmpf/rapidsmpf/tests/streaming/test_table_chunk.py

```diff
@@ -197,3 +197,18 @@ def test_spillable_messages(context: Context, stream: Stream) -> None:
     df2_got = df2_got.make_available_and_spill(context.br(), allow_overbooking=True)
     assert_eq(df2, df2_got.table_view())
     assert sm.get_content_descriptions() == {}
+
+
+def test_spillable_messages_by_context(context: Context, stream: Stream) -> None:
+    seq = 42
+    df1 = random_table(1024)
+
+    context.spillable_messages().insert(
+        Message(seq, TableChunk.from_pylibcudf_table(df1, stream, exclusive_view=True))
+    )
+    assert context.spillable_messages().get_content_descriptions() == {
+        0: ContentDescription(
+            content_sizes={MemoryType.DEVICE: 1024, MemoryType.HOST: 0},
+            spillable=True,
+        )
+    }
```

Comment:
Maybe just one more line to extract the message?

Side question: If I wanted to pull a specific sequence-id out of the collection, would I need to store the `sequence_id: mid` mapping separately? I'm not sure if this will ever come up, but was just curious.